flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/util/JobConfUtils.java

```diff
@@ -57,4 +57,10 @@ public static void addCredentialsIntoJobConf(JobConf jobConf) {
                     "Unable to add credentials into job conf when existing credentials in UGI.", e);
         }
     }
+
+    public static JobConf createJobConfWithCredentials(HiveConf hiveConf) {
```

Comment:
The parameter can be just `Configuration` instead of `HiveConf`

Reply:
Fixed.